tests/Model/NoDriverManagerTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\NewsBundle\Tests\Model;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\NewsBundle\Exception\NoDriverException;
+use Sonata\NewsBundle\Model\NoDriverManager;
+
+class NoDriverManagerTest extends TestCase
+{
+    /**
+     * @dataProvider providerMethods
+     */
+    public function testException($method, array $arguments)
+    {
+        $this->expectException(NoDriverException::class);
+        \call_user_func_array([new NoDriverManager(), $method], $arguments);
+    }
+
+    public function providerMethods()
```

Comment:
Either `provideMethods` or `methodsProvider` IMO

Reply:
It's just a copy and past 
https://github.com/sonata-project/SonataMediaBundle/blob/3.x/tests/Model/NoDriverManagerTest.php#L32